fastlane/Fastfile.swift

```diff
@@ -68,4 +68,20 @@ class Fastfile: LaneFile {
         Symbol.downloadFromAppStore(versionNumber: versionNumber, buildNumber: buildNumber)
         Symbol.uploadAppStoreToCrashlytics(versionNumber: versionNumber, buildNumber: buildNumber)
     }
+    
+    // MARK: - Register device
+
+    func registerNewDeviceLane() {
+        let deviceName = prompt(text: "Enter the device name")
+        let deviceUDID = prompt(text: "Enter the device UDID:")
+
+        registerDevice(
+            name: deviceName,
+            udid: deviceUDID,
+            teamId: .userDefined(Constant.teamId)
+        )
+
+        Match.syncCodeSigning(type: .development, appIdentifier: [], isForce: true)
+        Match.syncCodeSigning(type: .adHoc, appIdentifier: [], isForce: true)
```

Comment:
can we provide user the option to decide if they want automatic renew of provisioning profiles or not?

Reply:
@Shayokh144 Because we are registering new devices, so using `isForce: true` here to regenerate provisioning profiles. Normally, we don't need to use `isForce` :D.

But I will create other lanes for developers to run sync code signing.